cpp/ps0000/cps0000_Solution.cpp

```diff
@@ -1,7 +1,8 @@
-@include <"iostream">
+#include <"iostream"> // 1/2
 
-int Main()
+int main() // 0/1
 {
-  std::cout << "Hello, World!\n"<<endl;
-  return 0;
+  std::cout << "Hello, World!" << endl; // 4/4

```

Comment:
- [x] used `<<` for outputting instead of `>>`
- [x] used `\n` or `std::endl` in the right way
- [x] avoided syntax error by typing `;` at the end of the line
- [x] output is as required `Hello, World!` in a single line